src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
 // write code here
+const divs = document.querySelectorAll('.field');
```

Comment:
make more meaningful names
```suggestion
const fields = document.querySelectorAll('.field');
```